src/stare/models/analysis.py

```
@@ -132,3 +137,127 @@ def _coerce_extra_metadata(cls, data: Any) -> Any:
             )
             data["extraMetadata"] = {}
         return data
+
+    def __rich__(self) -> Panel:
```

Ruff (0.15.11)

[warning] Too many branches (22 > 12)

(PLR0912)